libdevcore/Log.h

```diff
@@ -73,10 +73,15 @@ BOOST_LOG_INLINE_GLOBAL_LOGGER_CTOR_ARGS(g_debugLogger,
     (boost::log::keywords::severity = 0)(boost::log::keywords::channel = "debug"))
 #define cdebug LOG(dev::g_debugLogger::get())
 
-BOOST_LOG_INLINE_GLOBAL_LOGGER_CTOR_ARGS(g_noteLogger,
+BOOST_LOG_INLINE_GLOBAL_LOGGER_CTOR_ARGS(g_note2Logger,
     boost::log::sources::severity_channel_logger_mt<>,
-    (boost::log::keywords::severity = 1)(boost::log::keywords::channel = "note"))
-#define cnote LOG(dev::g_noteLogger::get())
+    (boost::log::keywords::severity = 2)(boost::log::keywords::channel = "note2"))
+#define cnote LOG(dev::g_note2Logger::get())
+
+BOOST_LOG_INLINE_GLOBAL_LOGGER_CTOR_ARGS(g_note3Logger,
+    boost::log::sources::severity_channel_logger_mt<>,
+    (boost::log::keywords::severity = 3)(boost::log::keywords::channel = "note3"))
+#define cnote3 LOG(dev::g_note3Logger::get())
```

Comment:
I'd like to avoid additional global loggers. I'd like to minimize them in general and maybe will remove `cnote` at some point, too.

Global loggers here pollute `dev` namespace and they have an overhead for being thread-safe. It's better to use loggers more local to the task

Reply:
I think in this case I could just remote those debug traces from blockchain